src/components/molecules/LiveSchedule/LiveSchedule.scss

```diff
@@ -45,7 +45,7 @@
     &:nth-child(odd) {
       background-color: rgba($black, 0.3);
     }
-    &.schedule-event-container_live {
+    &.schedule-event-container--live {
```

Comment:
This is legacy style logic and I didn't want to mess with it

Reply:
Makes sense